game.rb

```diff
@@ -3,15 +3,30 @@
 
 class Game
 
-  def initialize io_handler, prompter
+  attr_reader :word, :letter
+
+  def initialize game_view, io_handler, prompter, validator
+    @game_view = game_view
     @io_handler = io_handler
     @prompter = prompter
+    @validator = validator
   end
 
   def game_handler
```

Comment:
Your method names should be verbs. This would probably be more understandable as something like `play_game`